### ISSUE TITLE
[Feature suggestion] Markdown formatting

### ISSUE THREAD

**Issue by DigiBear** [zombieFox/awesomeSheet#139]
For the large boxes, especially at the feats and abilities section, it would be great to do some basic text formatting to make large amounts of texts a bit easier to read.

**Comment by zombieFox** [comment]
Agreed, that would be good. I've looked into this in the past and it's a tall order as the logical progression here is to support Markdown. I found it difficult to find a workable solution the last time I tried to build this out. So I may resort to using an off the shelf solution/plugin. 

I'll add this to my project board and investigate in the future.

**Comment by johnantoni** [comment]
Quill could work https://quilljs.com/
It's a very light rich-text editor that saves as html, which could then be used to export as markdown.

Could look at integrating?

**Comment by zombieFox** [comment]
Another good suggestion. Thanks, I'm taking a look at quilljs now.

I would welcome any help with this issue.

**Comment by johnantoni** [comment]
I'll try. I've seen a couple of areas I can help out with. I'll see what I can do.

**Comment by johnantoni** [comment]
Nice pick @hexus i also like that it includes i18n support, something that is sadly lacking in Quill.
